bwtv.py

```diff
@@ -14,8 +14,10 @@
 from passlib.hash import apr_md5_crypt, sha512_crypt
 from requests import Session
 
-
-CONFIG_PATH = expanduser("~/.bw_teamvault_secrets.cfg")
+if "BW_TEAMVAULT_SECRET" in environ:
+    CONFIG_PATH = expanduser(environ.get("BW_TEAMVAULT_SECRET"))
+else:
+    CONFIG_PATH = expanduser("~/.bw_teamvault_secrets.cfg")
```

Comment:
This can be shortened to: 

```python
CONFIG_PATH = expanduser(environ.get("BW_TEAMVAULT_SECRET", "~/.bw_teamvault_secrets.cfg"))
```

Reply:
Also, BW_TEAMVAULT_SECRETS_FILE might be a better name as it indicates that the env var doesn't hold the actual secret.